plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsVmExecutorService.java

```diff
@@ -38,6 +39,9 @@
  */
 class CpsVmExecutorService extends InterceptingExecutorService {
 
+    private static final String QUEUE_SIZE_METRIC = "jenkins.job.pipeline.cps-vm-thread.queue.size";
+    private static final String EXECUTION_DURATION_METRIC = "jenkins.job.pipeline.cps-vm-thread.queue.size";
```

Comment:
Bad copy/paste when I created the constants:

```suggestion
    private static final String EXECUTION_DURATION_METRIC = "jenkins.job.pipeline.cps-vm-thread.execution.duration";
```